custom_components/tahoma/__init__.py

```diff
@@ -129,6 +135,29 @@ async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry):
             hass.config_entries.async_forward_entry_setup(entry, platform)
         )
 
+    async def handle_execute_command(call):
+        """Handle execute command service."""
+        entity_registry = await hass.helpers.entity_registry.async_get_registry()
+        entity = entity_registry.entities.get(call.data.get("entity_id"))
+        await tahoma_coordinator.client.execute_command(
+            entity.unique_id,
+            Command(call.data.get("command"), call.data.get("args")),
+            "Home Assistant Service",
```

Comment:
Why not just call the execute_command method we already have in place?

Reply:
cause `AttributeError: 'RegistryEntry' object has no attribute 'async_execute_command'`